resources/zib/ext-TimeInterval.Period.xml

```diff
@@ -14,7 +14,7 @@
       <use value="work" />
     </telecom>
   </contact>
-  <description value="An extension to provide all concepts of zib TimeInterval." />
+  <description value="A time interval indicates the time between two moments in time. Interval can be defined by a start time and end time, start time and duration or duration and end time." />

```

Comment:
In most other extensions we use a description of the form 'An extension to capture [description in zib]' or 'An extension to provide [description in zib]' or 'The xxx concept of zib yyy.' Not really that important, but we should consider to make these consistent later on (and add this to the profiling guidelines as well).

Reply:
You're right, and it's even documented:
> For extensions:
>    A description of what the extension is for.

I'll adjust this.